src/content/en/2021/privacy.md

```diff
@@ -212,76 +231,78 @@ Sensors can be useful to make a website more interactive, but could also be abus
-  description="Bar chart showing the percentage of websites that use the enumerateDevices() method to obtain information about the connected media devices.",
-  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=1398654768&format=interactive",
+  caption="Percent of desktop pages that used the `MediaDevicesEnumerateDevices` API.",
+  content="7.23%",
+  classes="big-number",
   sheets_gid="2141743069",
   sql_file="nb_websites_with_mediadevices_blink_usage.sql"
-  )
+)
 }}
 
-On 7.23% of websites, the `enumerateDevices()` method is called, which provides a list of the connected input devices.
+On 7.23% of desktop websites, and 5.33% of mobile websites the `enumerateDevices()` method is called, which provides a list of the connected input devices.
 
 ### Geolocation-as-a-service
 
-Geolocations services provide GPS and other location data of the user (such as <a hreflang="en" href="https://developer.mozilla.org/en-US/docs/Glossary/IP_Address">IP address</a>) and can be used by trackers to provide more relevant content to the user among other things.
-Therefore, we analyze the use of geolocation-as-a-service technologies on websites, based on libraries detected through [Wappalyzer](./methodology#wappalyzer).
+Geolocations services provide GPS and other location data of the user (such as [IP address](https://developer.mozilla.org/en-US/docs/Glossary/IP_Address)) and can be used by trackers to provide more relevant content to the user among other things. Therefore, we analyze the use of "geolocation-as-a-service" technologies on websites, based on libraries detected through [Wappalyzer](./methodology#wappalyzer).
 
 {{ figure_markup(
   image="nb_websites_using_each_geolocation.png",
   caption="Percentage of websites that use geolocation services",
-  description="Chart that shows the percentage of websites using each of the geolocation service libraries.",
+  description="Chart that shows the percentage of websites using each of the geolocation service libraries. ipify is used by 0.09% of desktop sites and 0.07% of mobile sites, MaxMind by 0.03% and 0.02% respectively, db-ip by 0.01% and 0.01%, and finally ipstack by 0.01% and 0.01%.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=2115506749&format=interactive",
   sheets_gid="1274602607",
   sql_file="nb_websites_using_each_geolocation.sql"
   )
 }}
 
-We find that the most popular service, <a hreflang="en" href="https://www.ipify.org/">ipify</a>, is used on 0.09% of desktop websites and 0.07% of mobile websites.
+We find that the most popular service, <a hreflang="en" href="https://www.ipify.org/">ipify</a>, is used on 0.09% of desktop websites and 0.07% of mobile websites. So, it would appear that few websites utilize geolocation services.
```

Comment:
```suggestion
We find that the most popular service, <a hreflang="en" href="https://www.ipify.org/">ipify</a>, is used on 0.09% of desktop websites and 0.07% of mobile websites. So, it would appear that few websites use geolocation services.
```